Add Security views to APM service page (#15846)
DataDog/documentation: content/en/tracing/services/service_page.md

The service page features summary cards with highlights on your service health. Easily spot potential faulty deployments, click into the card to view details or traces of the latest deployment, or view all deployments on this service. See new issues flagged on your service through our integration with [Error Tracking][4], where errors are automatically aggregated into issues.

{{< img src="tracing/visualization/service/summary_cards.png" alt="Summary cards" style="width:100%;">}}

> Our [Service Level Objectives (SLOs)][5] and [Incidents][6] summaries allow you to monitor the status of SLOs and ongoing incidents, so that you can keep performance goals top of mind. Click the cards to create a new SLO on the service or declare an incident. The [security signals][18] summary cards highlights how your services reacted to application threats.

Reviewer: ```suggestion
Our [Service Level Objectives (SLOs)][5] and [Incidents][6] summaries allow you to monitor the status of SLOs and ongoing incidents, so that you can keep performance goals top of mind. Click the cards to create a new SLO on the service or declare an incident. The [security signals][18] summary highlights how your services react to application threats.
```

{{< img src="tracing/visualization/service/error_tracking_side_panel.jpg" alt="Error Tracking tab"  style="width:90%;">}}

### Security

> View security signals on your service, which are automatically created when Datadog detects application attacks impacting your services, identifying meaningful threats to review instead of assessing each individual attack attempt. Read more about [Application Security][18].

Reviewer: ```suggestion
View security signals on your service, which are automatically created when Datadog detects application attacks impacting your services. The signals identify meaningful threats for you review instead of assessing each individual attack attempt. Read more about [Application Security][18].
```

> This tab has overview graphs that shows the timeline of attacks, the types of attacks, as well as the attacker information (Client IP or authenticated user). Click on a security signal to get information about what the detected threat is, and what actions can be taken to remediate it.

Reviewer: ```suggestion
This tab has overview graphs that show the timeline of attacks, the types of attacks, and attacker information (Client IP or authenticated user). Click on a security signal to get information about what the detected threat is and what actions you can take to remediate it.
```
